test/test_jit_fuser_te.py

```diff
@@ -338,6 +338,29 @@ def fn(s, x, y, z):
         ge = self.checkScript(fn, inputs)
         self.assertAllFused(ge.graph_for(*inputs))
 
+    @unittest.skipIf(not RUN_CUDA, "fuser requires CUDA")
```

Comment:
Nit: this test doesn't have to require cuda I think.

Reply:
Oh good call, it defaults to cpu-only.  I was half-smart about this one :-D